### PR TITLE
Some minor changes. A new loss function

### PR DESCRIPTION
**Pull request by jammy270**
Contains the following changes:

- Added a new loss function: expm1 = exp(x) -1 for the MAE loss. Switched to this new loss.
- Added a new configs directory to store the new config files and the ROI csv files. 
-  Added choices to the following args: rescale,arch,optimizer,lr_sch,norm. This provides a list to choose from and if the option doesn't match, it will throw an error.
- Set tensorflow logging level to display only warnings and errors.
- Set optimizer default to Adam (was RMSProp)
- Some other minor changes.

### CODE REVIEW

**Comment by gurayerus** [comment]
PR reviewed
